dffml/cli/ml.py

```diff
@@ -10,11 +10,15 @@
     KeysCMDConfig,
 )
 from ..base import config, field
+from ..accuracy import AccuracyScorer, MeanSquaredErrorAccuracy
 
 
 @config
 class MLCMDConfig(SourcesCMDConfig, ModelCMDConfig):
-    pass
+    accuracy_scorer: AccuracyScorer = field(
+        "Method to use to score accuracy",
+        default_factory=lambda: MeanSquaredErrorAccuracy(),
+    )
```

Comment:
A few things I realized

- Since this will only be used for the `accuracy` command, we should make a separate `Config` for it.
- Since we won't provide a default for the function in `high_level` we probably shouldn't define it here, that would create an inconsistency
- We should probably shorten the name of the flag to `-scorer`, since that avoid the user having to type out `-accuracy` twice.

```suggestion
    pass

@config
class AccuracyCMDConfig(MLCMDConfig):
    scorer: AccuracyScorer = field(
        "Method to use to score accuracy",
    )
```